docs/pages/overview.md

```diff
@@ -10,7 +10,7 @@ Meshery provides this high-level functionality:
 
 1. Service Mesh Performance Management
 1. Service Mesh Configuration Management
-    1. Configuration best practices
```

Comment:
This was not previously a numbered list, because it is a list of one. A bullet point might be more appealing. If we move to numbers, can we put a few more items in the list? Maybe take some from Meshery roadmap in the team drive?